raiden-ts/src/path/types.ts

```diff
@@ -68,3 +68,43 @@ export interface RaidenPFS {
   price: BigNumberish;
   token: string;
 }
+
+/**
+ * An IOU used to pay the services
+ */
+export const IOU = t.readonly(
+  t.type({
+    sender: Address,
+    receiver: Address,
+    amount: UInt(32),
+    // eslint-disable-next-line @typescript-eslint/camelcase
+    expiration_block: UInt(32),
+    // eslint-disable-next-line @typescript-eslint/camelcase
+    one_to_n_address: Address,
+    // eslint-disable-next-line @typescript-eslint/camelcase
+    chain_id: UInt(32),
+    signature: Signature,
```

Comment:
Careful. This is a required property, but in some places you initialize IOUs with type assertions without it. This is a problem, as you tell TypeScript to trust you about something, but it's not true.
Instead, I'd say you should use `Signed` type. For it, move `Signed` type from `messages/types.ts` to `utils/types.ts`, remove the `extends Message` constraint on the type, so it's valid for any type, and then use the distinction between `IOU` (not signed) and `Signed<IOU>` & `Signed(IOU)`.